### PR TITLE
[WIP] Update some pylab examples to OO format

### PR DESCRIPTION
**Pull request by dmgt**
## PR Summary

This supercedes #8892, and rewrites the Simple Plot example to use the now recommended object oriented syntax instead of the pyplot API

## PR Checklist
- [x] Code is PEP 8 compliant

### CODE REVIEW

**Comment by dmgt** [comment]
@choldgraf ready for you to review when you are ready

**Comment by choldgraf** [comment]
Hehe - ok so now it is giving the opposite error as before (now it's saying we have a white space at the end). Let me try and debug this and I will get back to you :-)

File: examples/pylab_examples/simple_plot.py [hunk 1]
```diff
@@ -7,13 +7,18 @@
...
 t = np.arange(0.0, 2.0, 0.01)
 s = 1 + np.sin(2*np.pi*t)
-plt.plot(t, s)
 
-plt.xlabel('time (s)')
-plt.ylabel('voltage (mV)')
-plt.title('About as simple as it gets, folks')
-plt.grid(True)
-plt.savefig("test.png")
+# Note that using plt.subplots below is equivilent to using
```
Comment (QuLogic) [inline examples/pylab_examples/simple_plot.py]: equivalent

**Comment by choldgraf** [comment]
hey @dmgt, it looks like the only thing left is that one type-o...let me know when you fix that up and then I'll merge!

**Comment by dmgt** [comment]
Thanks @choldgraf and @NelleV for reviewing this,  I didn't expect it to be merged so fast! 
Is it an issue that GitHub is now showing a Travis CI error for 95a8d39 above? 
There is also one more example I'd rewritten to OO format and will now add in a separate PR.

**Pull request merged**